irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -396,7 +396,7 @@ namespace iroha {
             SELECT CASE
                 WHEN EXISTS (SELECT * FROM inserted) THEN 0
                 %s
-                ELSE 5
+                ELSE 4
```

Comment:
It is absolutely unclear what are the magic numbers are used :( 
Even more - neither executor itself nor its methods do not have documentation.

Reply:
Error codes concept is still in development as I understand, so this is internal API.